### PR TITLE
First GUI implementation.

### PR DESCRIPTION
**Pull request by valentinbreiz**
This will provide you a screen manager with a desktop where you will can draw forms!

To do:
- [x] Fix font problem
- [x] Image rendering
- [ ] VBE Driver (fix memory leaks? problems)
- [x] Screen manager (init/update/render)
- [ ] Desktop (taskbar, basic menu, wallpaper, mouse) [WIP]

### CODE REVIEW

**Comment by WinMister332** [comment]
As stated in Slack and again this is conceptual and my opinion. But 3 different UI designs should be implemented, a text-based UI for servers, a basic GUI for low-end PCs or even Rasperry Pi, and Hi-Res/HD for hi-end PCs.
In my opinion both the basic and advanced GUI's should be fairly modern in design. You could even go as far to add sidebars.

You could even allow the user to swap between the basic and advanced GUIs. You could also make the basic UI an add-on for server environments. But, as stated previously all this is MY opinion and you're welcome to use any of these ideas, i'll even try my best to help program the UI's but I should warn you I'm studying Web-Design not C# or even COSMOS for that matter. So, I'm not sure how useful I would be in this part of the project.

**Comment by valentinbreiz** [comment]
Hello again @WinMister332. I think you're right, indeed at the beginning of the project with @geomtech we wanted to make 2 versions of Aura, Aura server in console mode, and Aura. The server edition will be only in console mode but the original version will be in console OR gui. The user will choose in the installation menu and after in settings.

**Comment by WinMister332** [comment]
It's understandable but there are some times when users would want a GUI for servers, so as I mentioned an add-on would be useful. It could be installed by your own homemade version of APT-GET or YUM.

File: Aura Operating System/Aura OS/System/GUI/UI/Desktop.cs [hunk 1]
```diff
@@ -19,11 +19,11 @@ class Desktop
     {
 
         public static VbeScreen Screen = new VbeScreen();
-        public static Canvas Canvas = new Canvas(Widht, Height);
+        public static Canvas Canvas = new Canvas(800, 600);
         public static SdfFont terminus;
 
-        public static int Widht = 800;
-        public static int Height = 600;
+        public static int Widht;
```
Comment (WinMister332) [inline Aura Operating System/Aura OS/System/GUI/UI/Desktop.cs]: You spelled "Width" wrong buddy.

Comment (valentinbreiz) [reply Aura Operating System/Aura OS/System/GUI/UI/Desktop.cs]: indeed x)

**Comment by valentinbreiz** [comment]
Current progress:
virtualbox_aura_15_03_2018_22_37_21

**Comment by MyvarHD** [comment]
your having trouble with the ppm performance ? is that all, anything else that need optimization please let me know.

**Comment by valentinbreiz** [comment]
Yes ppm rendering is very slow. I have some other bugs but I'll tell you about it on slack.

**Comment by prepare** [comment]
Hello all

This may help your GUI system.
visit => [CosmosOS RenderSurface Developing]( https://github.com/PaintLab/PixelFarm/issues/31)

**Pull request merged**